manager/integration/tests/test_metric.py

```diff
@@ -0,0 +1,159 @@
+import pytest
+import requests
+from prometheus_client.parser import text_string_to_metric_families
+from common import client, core_api, volume_name  # NOQA
+from common import get_self_host_id
+from common import create_pv_for_volume, create_pvc_for_volume
+from common import create_and_check_volume, wait_for_volume_healthy
+from common import write_volume_random_data
+from common import wait_for_volume_detached, wait_for_volume_degraded
+from common import wait_for_volume_faulted, crash_replica_processes
+from common import Mi
+from common import wait_for_volume_detached_unknown
+from common import LONGHORN_NAMESPACE
+
+longhorn_volume_state = {
```

Comment:
Can we instead generate these dynamically and use integers instead of floats?
i.e

`longhorn_volume_state = {vol_state:index for index,vol_state in enumerate(["creating","attached","detached","attaching","detaching","deleting"],1)}`

Reply:
Updated

Reply:
From my perspective, creating this dynamically seems to be unnecessarily complicating the code. @nitendra-suse , could you provide more insight into why dynamic generation is beneficial here?

Reply:
@c3y1huang I feel dynamical version is idiomatic python, But I agree your implementation is more straightforward, we can keep it.